src/packages/@ncigdc/containers/explore/presetFacets.ts

```diff
@@ -116,14 +116,157 @@ export const clinicalFacets: IClinicalFacetProps[] = [
+    'cog_renal_stage',
+    'cog_rhabdomyosarcoma_risk_group',
+    'enneking_msts_grade',
+    'enneking_msts_metastasis',
+    'enneking_msts_stage',
+    'enneking_msts_tumor_site',
+    'esophageal_columnar_dysplasia_degree',
+    'esophageal_columnar_metaplasia_present',
+    'figo_stage',
+    'inpc_grade',
+    'inpc_histologic_group',
+    'inrg_stage',
+    'inss_stage',
+    'irs_group',
+    'irs_stage',
+    'ishak_fibrosis_score',
+    'iss_stage',
+    'lymph_nodes_positive',
+    'lymph_nodes_tested',
+    'medulloblastoma_molecular_classification',
+    'metastasis_at_diagnosis',
+    'metastasis_at_diagnosis_site',
+    'micropapillary_features',
+    'mitosis_karyorrhexis_index',
+    'morphology',
+    'tumor_grade',
+    'tumor_regression_grade',
+    'wilms_tumor_histologic_subtype',
+    'tissue_or_organ_of_origin',
+    'site_of_resection_or_biopsy',
+    'circumferential_resection_margin',
+    'laterality',
+    'supratentorial_localization',
+    'gastric_esophageal_junction_involvement',
+    'lymphatic_invasion_present',
+    'perineural_invasion_present',
+    'peripancreatic_lymph_nodes_positive',
+    'tumor_confined_to_organ_of_origin',
+    'tumor_focality',
+    'tumor_largest_dimension_diameter',
+    'vascular_invasion_present',
+    'vascular_invasion_type',
+    'best_overall_response',
+    'progression_or_recurrence',
+    'residual_disease',
+  ],
+  'diagnoses.treatments': [
+    'days_to_treatment_start',
+    'days_to_treatment_end',
+    'therapeutic_agents',
+    'initial_disease_status',
+    'treatment_anatomic_site',
+    'treatment_effect',
+    'treatment_intent_type',
+    'treatment_or_therapy',
+    'treatment_outcome',
+    'treatment_type',
+  ],
+  exposures: [
+    'alcohol_history',
+    'alcohol_intensity',
+    'asbestos_exposure',
+    'radon_exposure',
+    'tobacco_smoking_status',
+  ],
+  follow_ups: [
+    'adverse_event',
+    'barretts_esophagus_goblet_cells_present',
+    'comorbidity',
+    'comorbidity_method_of_diagnosis',
+    'diabetes_treatment_type',
+    'disease_response',
+    'ecog_performance_status',
+    'hepatitis_sustained_virological_response',
+    'hpv_positive_type',
+    'karnofsky_performance_status',
+    'menopause_status',
+    'progression_or_recurrence',
+    'progression_or_recurrence_anatomic_site',
+    'progression_or_recurrence_type',
+    'reflux_treatment_type',
+    'risk_factor',
+    'risk_factor_treatment',
+    'viral_hepatitis_serologies',
+    'bmi',
+    'height',
+    'weight',
+  ],
+  molecular_test: [
+    'antigen',
+    'biospecimen_type',
+    'chromosome',
+    'gene_symbol',
+    'histone_family',
+    'histone_variant',
+    'laboratory_test',
+    'mismatch_repair_mutation',
+    'molecular_consequence',
+    'ploidy',
+    'second_gene_symbol',
+    'test_analyte_type',
+    'variant_type',
+    'zygosity',
+  ],
+}
+
+export const customSorting = (field: string = ''): object | number => {
```

Comment:
This is where the magic happens. Pretty much every other modified file is just formatting.